src/main/java/org/purescript/PSLanguage.kt

```diff
@@ -22,5 +22,22 @@ class PSLanguage : Language("Purescript", "text/purescript", "text/x-purescript"
             "Prim.Symbol",
             "Prim.TypeError",
         )
+
+        /**
+         * These types are built into the purescript compiles,
+         * and are always available.
+         *
+         * See [https://pursuit.purescript.org/builtins/docs/Prim] for details.
+         */
+        val BUILTIN_TYPES = setOf(
+            "Int",
+            "Number",
+            "String",
+            "Char",
+            "Boolean",
+            "Array",
+            "Type", // TODO Type is really a kind, not a type
+            "Row", // TODO Row is really a kind, not a type
+        )
```

Comment:
I am not 100 % sure that this set is exhaustive. I spent some time searching, and only found these.

`Type` and `Row` are built-in, but they are not types. When we get better support for kinds, we should improve this.